scripts/components/App.js

```diff
@@ -42,32 +96,33 @@ export default class App extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = this.props.store.state;
-    this.props.store.on("change", state => {
-      this.setState(Object.assign({busy: false}, state));
-    });
-    this.props.store.on("error", error => {
-      this.setState({busy: false, error: error.message});
-    });
-    this.props.store.load();
+    this.state = {busy: false, error: "", items: []};
```

Comment:
In the constructor, I can assign the state !

Look https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#es6-classes